scss/civicrm/custom-civicrm.scss

```diff
@@ -5,21 +5,22 @@
 @import 'SCSSROOT/org.civicrm.shoreditch/base/scss/overrides/variables';
 @import 'SCSSROOT/org.civicrm.shoreditch/base/scss/vendor/bootstrap/variables';
 @import 'SCSSROOT/org.civicrm.shoreditch/base/scss/vendor/bootstrap/mixins/ALL';
-@import "SCSSROOT/org.civicrm.shoreditch/base/scss/vendor/bootstrap/tooltip";
+@import 'SCSSROOT/org.civicrm.shoreditch/base/scss/vendor/bootstrap/tooltip';
 @import '../bootstrap/mixins/*';
 @import '../bootstrap/fonts/*';
 @import '../bootstrap/vendor/font-awesome/variables';
 @import '../bootstrap/vendor/font-awesome/path';
 @import '../bootstrap/overrides/style/*';
+@import '../angular/overrides/*';
```

Comment:
see the same comments i made in bootstrap.scss

Reply:
Same as before

Reply:
see https://github.com/compucorp/civihr-employee-portal-theme/pull/325#discussion_r174040994